doc/using_mxnet.rst

```diff
@@ -786,46 +695,113 @@ This uploads the contents of my_model to a gzip compressed tar file to S3 in the
+-  ``source_dir`` Path (absolute or relative) to a directory with any
+   other training source code dependencies including the entry point
+   file. Structure within this directory will be preserved when training
+   on SageMaker.
+-  ``dependencies (list[str])`` A list of paths to directories (absolute or relative) with
+   any additional libraries that will be exported to the container (default: ``[]``).
+   The library folders will be copied to SageMaker in the same folder where the entrypoint is copied.
+   If the ``source_dir`` points to S3, code will be uploaded and the S3 location will be used
+   instead. For example, the following call
 
-The Docker images have the following dependencies installed:
+   >>> MXNet(entry_point='train.py', dependencies=['my/libs/common', 'virtual-env'])
 
-+-------------------------+--------------+-------------+-------------+-------------+-------------+-------------+-------------+
-| Dependencies            | MXNet 0.12.1 | MXNet 1.0.0 | MXNet 1.1.0 | MXNet 1.2.1 | MXNet 1.3.0 | MXNet 1.4.0 | MXNet 1.4.1 |
-+-------------------------+--------------+-------------+-------------+-------------+-------------+-------------+-------------+
-| Python                  |   2.7 or 3.5 |   2.7 or 3.5|   2.7 or 3.5|   2.7 or 3.5|   2.7 or 3.5|   2.7 or 3.6|   2.7 or 3.6|
-+-------------------------+--------------+-------------+-------------+-------------+-------------+-------------+-------------+
-| CUDA (GPU image only)   |          9.0 |         9.0 |         9.0 |         9.0 |         9.0 |         9.2 |        10.0 |
-+-------------------------+--------------+-------------+-------------+-------------+-------------+-------------+-------------+
-| numpy                   |       1.13.3 |      1.13.3 |      1.13.3 |      1.14.5 |      1.14.6 |      1.16.3 |      1.14.5 |
-+-------------------------+--------------+-------------+-------------+-------------+-------------+-------------+-------------+
-| onnx                    |          N/A |         N/A |         N/A |       1.2.1 |       1.2.1 |       1.4.1 |       1.4.1 |
-+-------------------------+--------------+-------------+-------------+-------------+-------------+-------------+-------------+
-| keras-mxnet             |          N/A |         N/A |         N/A |         N/A |       2.2.2 |     2.2.4.1 |     2.2.4.1 |
-+-------------------------+--------------+-------------+-------------+-------------+-------------+-------------+-------------+
```

Comment:
Any special reason to remove this list of dependencies here?

Reply:
I believe this is info better left only in readme, so it doesn't have to be updated twice to stay in sync. I added a link to the section in the readme here in my latest commit.